packages/lib-user/src/components/UserHome/components/RecentSubjects/RecentSubjectsContainer.js

```diff
@@ -12,19 +13,50 @@ const SWROptions = {
   refreshInterval: 0
 }
 
+async function fetchUserRecents({ userId }) {
+  try {
+    const query = {
+      page_size: 10,
+      sort: '-created_at'
+    }
+    const response = await panoptes.get(`/users/${userId}/recents`, query)
+    return response.body?.recents
+  } catch (error) {
+    console.error(error)
+    throw error
```

Comment:
Similar to RecentProjectsContainer, since the related components handle error states I think this fetch should throw an error if it has one so that error can be included in the error returned from useSWR on line 35?